File: MATEMÁTICA/1585/main.c
#include <stdio.h>

int main() {
    int n, x, y, area;
    scanf("%d", &n);
    for (int times = 0; times < n; times++) {
        scanf("%d %d", &x, &y);
        area = x * y / 2;
        printf("%d cm2\n", area);
    }
    return 0;
}

File: MATEMÁTICA/1585/main.dart
import 'dart:io';

void main() {
  int n = int.parse(stdin.readLineSync()!);
  for (int times = 0; times < n; times++) {
    var [x, y] = stdin.readLineSync()!.split(' ').map(int.parse).toList();
    int area = x * y ~/ 2;
    print('${area} cm2');
  }
}
